### ISSUE TITLE
The "tab" key does not work for selecting/reordering resources

### ISSUE THREAD

**Issue by MindfulQA** [learningequality/kolibri#7643]
### Observed behavior

When the user presses the tab key, a resource is not selected. Instead, a button appears in the top left corner reading "skip to main content." Pressing this button closes it, but does not cause any other response.

![Screen Shot 2020-10-28 at 9 07 48 PM](https://user-images.githubusercontent.com/73143721/97524350-af63af80-1961-11eb-80d2-cff15eaa590a.png)

…

### Expected behavior

The user should be able to use the tab key and arrows on the keyboard to toggle between resources and reorder them.
…

### User-facing consequences

User would not be able to navigate via shortcuts as easily, and this could have a particularly severe impact on people who are blind, have motor disabilities, etc.
…

### Errors and logs

N/A
…

### Steps to reproduce

1. Open Kolibiri desktop app and log in to Coach account
2. Click on the "Plan" tab at the top of the page.
3. Click on the title of an existing lesson (that already contains resources).
4. Press the "tab" key on the keyboard.
5. Observe that a button appears in the top left corner reading "Skip to main content"
6. Click on the button 
7. Observe that the button disappears but nothing else happens.

Note for testing - issue occurs for the scenario in line 28:

https://github.com/learningequality/kolibri/blob/release-v0.14.x/integration_testing/features/coach/coach-lesson-manage-resources.feature

…

### Context

macOS Catalina 12.15.7 - Kolibri version 0.14
…

**Comment by jonboiser** [comment]
@MindfulQA there's a known issue #7651 where that green outline doesn't appear everywhere it should, but I just tried to reproduce this, and I'm not sure that's the problem here.

The "Skip to main content" button is something called a "skip-navigation link" that lets keyboard-only users avoid having to tab through the links in the navigation and get to the main part of the page.

I just tried this out on the https://kolibri-beta.learningequality.org, and I think this page is working as it should.

1. Start at the "skip to main content" button
1. Press "enter"
1. Press "tab" twice (ideally, you'd only need to do this once, so that's a legitimate issue).
1. Your focus should be on the "Options" button (it has the green outline).

I then tried reordering resources with the keyboard, and that seems to be working too. Are you losing keyboard focus from the drag handles?

**Comment by jonboiser** [comment]
I'm not able to reproduce this issue right now. I believe that this might be a misunderstanding stemming from the behavior that the skip link goes to the "Lesson title" header and not to the drag handles for resources.
